lib/cache/cache_test.go

```diff
@@ -245,14 +245,14 @@ func newPackWithoutCache(dir string, opts ...packOption) (*testPack, error) {
 	p.provisionerS = local.NewProvisioningService(p.backend)
 	p.eventsS = newProxyEvents(local.NewEventsService(p.backend), cfg.ignoreKinds)
 	p.presenceS = local.NewPresenceService(p.backend)
-	p.usersS = local.NewIdentityService(p.backend)
+	p.usersS = local.NewTestIdentityService(p.backend)
```

Comment:
Suggestion: Extract `local.NewTestIdentityService(p.backend)` and use it throughout?